api/server/structs/endpoints_lightclient.go

```diff
@@ -4,17 +4,51 @@ type LightClientHeader struct {
 	Beacon *BeaconBlockHeader `json:"beacon"`
 }
 
+type LightClientHeaderCapella struct {
+	Beacon          *BeaconBlockHeader             `json:"beacon"`
+	Execution       *ExecutionPayloadHeaderCapella `json:"execution"`
+	ExecutionBranch [][]byte                       `json:"execution_branch"`
```

Comment:
```suggestion
	ExecutionBranch [][]string                       `json:"execution_branch"`
```
We return hex-encoded values from the API